### ISSUE TITLE
Dev Notes

### ISSUE THREAD

**Issue by IDIDIR** [IDIDIR/IPM#2]
Yurk, write here when you see it

**Comment by IDIDIR** [comment]
accept invite for collaboration, please

**Comment by IDIDIR** [comment]
bro, you use VS or Atom?

**Comment by IDIDIR** [comment]
bro, you use the VS or the Atom?

**Comment by IDIDIR** [comment]
never don't use this lib `pycrypto`, I danced with bybin allday suka. 
blahblahblah
`Microsoft Visual C++ 14.0 is required.`
blahblahblah
`warning: GMP or MPIR library not found; Not building Crypto.PublicKey._fastmath.`
blahblahblah
`C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\inttypes.h(27): error C2061: бЁ-в ЄбЁз?бЄ п RиЁЎЄ : Ё¤?-вЁдЁЄ вRа "intmax_t"`
blahblahblah
error error error

**Comment by nanoqsh** [comment]
Where is invite?

**Comment by IDIDIR** [comment]
@nekosora I think here https://github.com/IDIDIR/IPM/invitations

**Comment by IDIDIR** [comment]
will we use double or single quotes?

**Comment by nanoqsh** [comment]
'

**Comment by IDIDIR** [comment]
@nekosora, register here, please https://appcenter.ms/apps and say, Is it freeware and does team chat work there?

**Comment by nanoqsh** [comment]
Sorry for the long absence, I accidentally opened vim

**Comment by nanoqsh** [comment]
nekosora
----------------------------------
move email to enc msgs, please (с) IDIDIR

**Comment by IDIDIR** [comment]
@nekosora Ohnoo, sensetive data must be encrypted with the IPM project. 
Clone the repository, please.

**Comment by nanoqsh** [comment]
The IPM doesn't have input data. Just displays:
`foo bazat blahblah`

**Comment by IDIDIR** [comment]
Thanks for the wish, I hope this will be realized soon!

**Comment by IDIDIR** [comment]
You can release the nightly build by uncommenting the first line. Thank you!

**Comment by nanoqsh** [comment]
Ok. I will wait an update

**Comment by IDIDIR** [comment]
need to read GitHub API for add the login and the parse functions

**Comment by nanoqsh** [comment]
Where is an update?

**Comment by IDIDIR** [comment]
@nekosora ooohhh, the last time in the room there is some kind of repair, and the hard drive and source code somewhere is packed. 
but don't packed the Khadas VIM2 Pro
![a1c7cece-efa3-4153-be48-784c87ab0684](https://user-images.githubusercontent.com/21132031/45922366-1fceb580-beec-11e8-86b4-a34fff7c3a04.jpg)

**Comment by nanoqsh** [comment]
Ok, I'll be waiting for you setup your vim

**Comment by IDIDIR** [comment]
yes, and while I setup the o'boys

**Comment by nanoqsh** [comment]
did you die?

**Comment by IDIDIR** [comment]
@nekosora nonono, bro. all this time I was working on a gist, bro
https://gist.github.com/IDIDIR/bc5f8681ef171eeadd63b07455bead83

**Comment by IDIDIR** [comment]
Bro, was developed a new startup, which will be laid out in the open source. he will just be cannon bro

**Comment by nanoqsh** [comment]
That's cool, I'm waiting it already

**Comment by IDIDIR** [comment]
@nekosora bro send to our senior [this link](https://github.com/IDIDIR/IPM/invitations)
without him in this conference we can not cope

**Comment by nanoqsh** [comment]
@IDIDIR ok. I will send

**Comment by IDIDIR** [comment]
@nekosora beta. check please.
token generation needed (in readme)

**Comment by onlineasd2** [comment]
What hapened ?

**Comment by IDIDIR** [comment]
@onlineasd2 why you still have not registered our AICC inc. here?

**Comment by IDIDIR** [comment]
@nekosora brooo, you can't imagine. from 2k17 to js zavezli async iterators
```
for await (const bazat of readLines(filePath)) {
  console.log(`hello, ${bazat}`)
}
```

**Comment by IDIDIR** [comment]
@nekosora minor upgrade

**Comment by IDIDIR** [comment]
@onlineasd2 bro, why is there still no our company?

**Comment by IDIDIR** [comment]
@nekosora ohh, this bro @AlanLinc did not see the era of paid private repositories

**Comment by IDIDIR** [comment]
@nekosora update is available
changes: https://github.com/IDIDIR/IPM/commit/0a321fc4207d349f2d2cb4cf25eb98b6b3db6c23
please test this, 
first send msg to your username + load self msg
and second testcase with my username

**Comment by IDIDIR** [comment]
so really security with personal keys!1

**Comment by IDIDIR** [comment]
@onlineasd2 bro, how are your ray tracing startups?

**Comment by onlineasd2** [comment]
I'll show on E3

**Comment by IDIDIR** [comment]
@onlineasd2 share with us your first impressions in the RSA chat.  https://github.com/IDIDIR/IPM/issues/3
Free & Secure !
Python & Free !

**Comment by IDIDIR** [comment]
@nekosora urgent, bro, ask financing from our financial director
https://domains.google.com/m/registrar/search?searchTerm=aicc.dev
after 28.02 - $12

**Comment by IDIDIR** [comment]
@nekosora https://allennlp.org/tutorials#annotated-code it's fantastic, bro

**Comment by nanoqsh** [comment]
annotated code?

**Comment by IDIDIR** [comment]
@nekosora yes, I could not even imagine that neural networks went so far ahead of comparison with typical developers.

**Comment by nanoqsh** [comment]
I don't understand anything

**Comment by IDIDIR** [comment]
@nekosora 
bro, if you look closely in this you can see the pain and four hours of my life
![image](https://user-images.githubusercontent.com/21132031/59138207-a90c1680-89ad-11e9-9411-f8231b5a2d48.png)

**Comment by nanoqsh** [comment]
@IDIDIR I have no idea what it is, but looks pretty nice 👍

**Comment by nanoqsh** [comment]
That's enough to perestavlat stars!

**Comment by IDIDIR** [comment]
@nanolsn parsing claims data requires strong encryption. please get encryption until tomorrow!
documentation: https://github.com/IDIDIR/IPM

best regards, IDIDIR

**Comment by nanoqsh** [comment]
I lost my keys

**Comment by IDIDIR** [comment]
@nanolsn it will restore all your private keys in the history of your career. just run her

**Comment by IDIDIR** [comment]
@nanolsn bro, can you imagine now bazat can move to silicon valley!!1
https://mesto.co/#about

**Comment by IDIDIR** [comment]
sorry for the interrupt, my electricity was cut off until recently

regards, IDIDIR

**Comment by nanoqsh** [comment]
Even without electricity you can write code
